extension.toml

```diff
@@ -0,0 +1,18 @@
+id = "sway"
+name = "Sway"
+description = "Sway smart contract language support with LSP integration"
+version = "0.1.0"
+schema_version = 1
+authors = ["Fuel Labs <contact@fuel.sh>"]
+repository = "https://github.com/FuelLabs/sway-zed-extension"
+
+[grammars.sway]
+repository = "https://github.com/FuelLabs/tree-sitter-sway"
+rev = "master"
```

Comment:
do we want to pin to specific tag instead (since master is a moving target)

Reply:
Good idea, I originally did this but changed it to master as I think it's better to always provide the latest tree-sitter syntax rather than needing to update a commit hash.